subprojects/core-api/src/main/java/org/gradle/api/configuration/BuildFeature.java

```diff
@@ -31,30 +31,43 @@
 public interface BuildFeature {
 
     /**
-     * Whether the feature was requested for the build.
+     * Status of the feature flag with three possibilities:
+     * <ul>
+     *     <li>true - requested to opt in
```

Comment:
❓ Would it also be true to say `true - the feature has been explicitly enabled by the user`?